articles/client-platforms/angularjs.md

```diff
@@ -6,9 +6,9 @@ image: //auth0.com/lib/platforms-collection/img/angular.png
 tags:
   - quickstart
 snippets:
-  dependancies: angularjs/dependancies.html
-  setup: angularjs/app.js
-  use: angularjs/login.js
+  dependancies: client-platforms/angularjs/dependancies
```

Comment:
We agreed not moving them into `client-platforms`, but actually to technology name / snippet. Why the change @ntotten ?